sample/build.gradle.kts

```diff
@@ -19,8 +19,11 @@ android {
     versionName = "1"
     testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     testApplicationId = "com.slack.circuit.sample.androidTest"
+
   }
 
+  buildFeatures { compose = true }
+
```

Comment:
I think this is redundant right? We have this in our root gradle confic

Reply:
Yup, looks like it is. Removed